src/modal/modal.js

```diff
@@ -413,14 +413,19 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.stackedMap'])
           backdrop: modal.backdrop,
           keyboard: modal.keyboard,
           openedClass: modal.openedClass,
-          windowTopClass: modal.windowTopClass
+          windowTopClass: modal.windowTopClass,
+          container: modal.container
         });
 
         openedClasses.put(modalBodyClass, modalInstance);
 
-        var body = $document.find('body').eq(0),
+        var container = $document.find(modal.container).eq(0),
             currBackdropIndex = backdropIndex();
 
+        if (!container.length) {
```

Comment:
We shouldn't mention jqLite because if the user includes the full jQuery library before Angular, Angular will use the full jQuery library.  This goes back to how we want to do error handling as mentioned above.